discord-bot/bot/config.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding=utf-8 -*-
+"""Configuration for the bot."""
+
+import logging
+from dataclasses import dataclass
+from os import getenv
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class Config:
```

Comment:
Could you please use a [pydantic settings](https://docs.pydantic.dev/usage/settings/) class here?